### DISCUSSION TITLE
빌드 세팅 관련해

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#988] [5. Q&A]
안녕하세요, 빌드세팅 관련해 질문드립니다.

현재 월드 용량은 260mb 즈음 되고 구동에 문제는 없으나 안드로이드 기종에서 특히 버벅임이 심합니다. 아이폰 환경에선 무리없이 작동이 되는데 이가 빌드 세팅에서 플랫폼이 ios 로 작업된 것과 연관이 있을지 여쭙고 싶습니다.

**Comment by ZdevMonZ** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/851 요거 참고하시면 될것 같아요

**Reply by esang28** [reply]
감사합니다 ^^

**Comment by YSHFM** [comment]
https://www.mettit.com/log/view/LOG20220711180904573 해당 링크도 참고해보세요

**Reply by esang28** [reply]
어우 3기가나 차지하고 있었네요. 정보 감사합니다^^

**Reply by YSHFM** [reply]
동일한 이슈있어서.... 저장공간 정리 하니 프레임 드랍이 사라졌습니다.
